.isort.cfg

```diff
@@ -1,7 +1,8 @@
 [settings]
-multi_line_output=3
-include_trailing_comma=True
 force_grid_wrap=0
-use_parentheses=True
-line_length=100
 force_sort_within_sections=True
+include_trailing_comma=True
+known_third_party=xain_proto,xain_sdk
```

Comment:
just for reference, can you just comment what the problem you had with `isort` was? did this fix it?

Reply:
my ide is set up such that it can find my local repos via the `PYTHONPATH` env (basically pointing python to the local xain-fl, xain-sdk, etc). it seems that pip is looking up that env as well next to the actual installed packages. therefor isort then thought that e.g. xain-sdk is a local dependency instead of a 3rd party dependency, which conflicted with the ci settings.

pointing out xain-sdk and xain-proto as known 3rd party dependencies to isort, the imports are getting sorted correctly no matter what is happening on the local system's env vars.

Reply:
good to know